### PR TITLE
ซูฟียาน

### PR DESCRIPTION
**Pull request by Sufiyang**
##[section]Starting: Run build

### CODE REVIEW

File: src/libstd/fs.rs [hunk 1]
```diff
@@ -113,6 +113,8 @@ pub struct Metadata(fs_imp::FileAttr);
 /// will yield instances of [`io::Result`]`<`[`DirEntry`]`>`. Through a [`DirEntry`]
 /// information like the entry's path and possibly other metadata can be
 /// learned.
+/// The order in which this iterator returns entries is platform and filesystem
```
Comment (hellow554) [inline src/libstd/fs.rs]: trailing whitespace: remove that please

File: src/libstd/fs.rs [hunk 2]
```diff
@@ -1955,6 +1957,8 @@ pub fn remove_dir_all<P: AsRef<Path>>(path: P) -> io::Result<()> {
 ///
 /// This function currently corresponds to the `opendir` function on Unix
 /// and the `FindFirstFile` function on Windows.
+/// The order in which this iterator returns entries is platform and filesystem
```
Comment (hellow554) [inline src/libstd/fs.rs]: trailing whitespace: remove that please

**Comment by hellow554** [comment]
Few things: 
  * please squash your two commits into one:  https://github.com/rust-lang/rust/blob/master/CONTRIBUTING.md#pull-requests
  * Why did you choose such a "weird" title? 
  * Why is the author of the commits a different one than you are? ping @ali-raheem

**Comment by ali-raheem** [comment]
I think this might be some sort of spam.

Currently working on this here https://github.com/rust-lang/rust/issues/63183

**Comment by Mark-Simulacrum** [comment]
I'm going to close this PR since it appears to be either accidental or, in any case, isn't really clear about it's intent.

**Pull request closed without merging**